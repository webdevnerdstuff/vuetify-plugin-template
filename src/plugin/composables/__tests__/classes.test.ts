import { describe, it, expect } from 'vitest';
import {
	useContainerClasses
} from '../classes';


describe('Classes Composable', () => {

	describe('useContainerClasses', () => {
		it('should return class object', () => {
			const classes = useContainerClasses({
				isOption: true,
			});

			expect(classes).toMatchSnapshot();
		});
	});
});
