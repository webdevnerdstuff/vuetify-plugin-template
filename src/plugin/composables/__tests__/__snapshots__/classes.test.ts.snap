// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Classes Composable > useContainerClasses > should return class object 1`] = `
{
  "v-plugin-template": true,
  "v-plugin-template--container": true,
  "v-plugin-template--container-option": true,
}
`;
